sqlhelper.py

```diff
@@ -0,0 +1,37 @@
+import sqlite3
+
+#database setup function
+def setup_database(dbfile):
+    conn = sqlite3.connect(dbfile)
+    c = conn.cursor()
+
+    c.execute('''CREATE TABLE IF NOT EXISTS garage_status
+                 (id INTEGER PRIMARY KEY AUTOINCREMENT,
+                  garage_names TEXT, 
+                  garage_fullness TEXT, 
+                  garage_addresses TEXT,
+                  timestamp TEXT DEFAULT CURRENT_TIMESTAMP)''')
```

Comment:
```suggestion
                  timestamp TEXT)''')
```
we will probably want to send the timestamp from the server so we know at what point we scraped it, not when we inserted it into the database